contracts/exchange-libs/contracts/src/LibEIP712.sol

```diff
@@ -21,54 +21,65 @@ pragma solidity ^0.5.5;
 
 contract LibEIP712 {
 
-    // EIP191 header for EIP712 prefix
-    string constant internal EIP191_HEADER = "\x19\x01";
-
-    // EIP712 Domain Name value
-    string constant internal EIP712_DOMAIN_NAME = "0x Protocol";
-
-    // EIP712 Domain Version value
-    string constant internal EIP712_DOMAIN_VERSION = "2";
-
     // Hash of the EIP712 Domain Separator Schema
     bytes32 constant internal EIP712_DOMAIN_SEPARATOR_SCHEMA_HASH = keccak256(abi.encodePacked(
         "EIP712Domain(",
         "string name,",
         "string version,",
+        "uint256 chainId,",
         "address verifyingContract",
```

Comment:
Let's also take this opportunity to rename `verifyingContract` to `verifyingContractAddress` for consistency.

Reply:
You sure? Seems like that [breaks spec](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md#definition-of-domainseparator).

Reply:
The spec is purely an example, we can technically add any fields that we want. I just don't like that we're calling this field 2 different things throughout the codebase.